priv/repo/migrations/20200831011352_set_messages_seen_at.exs

```diff
@@ -0,0 +1,44 @@
+defmodule ChatApi.Repo.Migrations.SetMessagesSeenAt do
+  use Ecto.Migration
+
+  import Ecto.Query, warn: false
+
+  alias ChatApi.{Messages, Repo}
+  alias ChatApi.Conversations.Conversation
+  alias ChatApi.Messages.Message
+
+  def up do
```

Comment:
May want to look into `change/0` which manages both up and down migrations without having to explicitly writing them out in your migration: https://hexdocs.pm/ecto_sql/Ecto.Migration.html#module-change

Reply:
i typically use `change/0`, but in this case i thought it wouldn't work 🤔 i'll give it a shot though!

Reply:
(didn't work in this case -- would've worked if i had done this in the last migration where i added the `seen_at` field in the first place... oops 😅)

Reply:
No worries, makes sense! 😄